simulation/main/Simulation.py

```diff
@@ -237,10 +239,10 @@ def run_model(self, speed=np.array([20, 20, 20, 20, 20, 20, 20, 20]), plot_resul
         if plot_results:
             results_arrays = self.get_results(["speed_kmh", "distances", "state_of_charge", "delta_energy",
                                                "solar_irradiances", "wind_speeds", "gis_route_elevations_at_each_tick",
-                                               "cloud_covers", "raw_soc"]) + [raw_speed]
+                                               "cloud_covers", "raw_soc", "regen_produced_energy"]) + [raw_speed]
             results_labels = ["Speed (km/h)", "Distance (km)", "SOC (%)", "Delta energy (J)",
```

Comment:
I don't think it's necessary to have the regen_produced_energy plotted by default